src/components/cooperation-section-view/CooperationSectionView.tsx

```diff
@@ -20,8 +24,9 @@ const CooperationSectionView: FC<CooperationSectionViewProps> = ({ item }) => {
 
   const resources = useMemo<undefined | ReactNode[]>(
     () =>
-      item.resources?.map(({ resource, resourceType }) => (
+      item.resources?.map(({ availability, resource, resourceType }) => (
         <ResourceItem
+          availability={availability as ResourceAvailability}
```

Comment:
can we change `availability` in the `Resource` interface in the file `src/types/course/interfaces/course.interface.ts` to remove this `as ResourceAvailability`?
E. g.:
<img width="319" alt="Screenshot 2024-09-17 at 11 58 32" src="https://github.com/user-attachments/assets/6a0becb2-a81a-4e07-805d-681a179b0c79">

Reply:
Fixed